spec/support/acceptance/tar.rb

```diff
@@ -0,0 +1,37 @@
+begin
+  require 'serverspec'
+rescue LoadError
+  # Not using acceptance tests
+else
+  module Serverspec
+    module Type
+      class Tar < Command
+        def contents
+          command_result.stdout
+        end
+
+        def contents_count
+          command_result.stdout.split("\n").length
+        end
+
+        private
+
+        def command
+          "tar -tvf #{@name}"
+        end
+
+        def command_result
+          @command_result ||= @runner.run_command(command)
+        end
```

Comment:
Since https://github.com/mizzy/serverspec/commit/2e7e24a90522df8e435a689bd59d110ed56799d3 (included in serverspec 2.41.8) this is technically redundant. You may need to make `command` `protected` instead of `private`. However, we didn't change other ocurrances so I'm fine either way.